cypress/src/tests/e2eTests/verifyAuthUserCheckout.spec.js

```diff
@@ -1,197 +1,231 @@
-         */
-        /**
-         * TODO - when /customer/orders page will be ready
-         * Redirect to /customer/orders
-         * Confirm that new order is visible on Orders page
-         */
-    });
+    assertCartSummaryProduct(
+      'Hollister Backyard Sweatshirt',
+      'MH05',
+      '1',
+      '$52.00',
+      '$52.00',
+      '1'
+    )('.commerce-cart-wrapper');
+    assertTitleHasLink(
+      'Hollister Backyard Sweatshirt',
+      '/products/hollister-backyard-sweatshirt/MH05'
+    )('.commerce-cart-wrapper');
+    assertProductImage('/mh05-white_main_1.jpg')('.commerce-cart-wrapper');
+    cy.contains('Estimated Shipping').should('be.visible');
+    cy.get('.dropin-button--primary')
+      .contains('Checkout')
+      .click();
+    assertCartSummaryMisc(2);
+    assertCartSummaryProductsOnCheckout(
+      'Crown Summit Backpack',
+      '24-MB03',
+      '1',
+      '$38.00',
+      '$38.00',
+      '0'
+    );
+    assertCartSummaryProductsOnCheckout(
+      'Hollister Backyard Sweatshirt',
+      'MH05',
+      '1',
+      '$52.00',
+      '$52.00',
+      '1'
+    );
+    setGuestShippingAddress(customerShippingAddress, true);
+    uncheckBillToShippingAddress();
+    cy.wait(2000);
+    setGuestBillingAddress(customerBillingAddress, true);
+    assertOrderSummaryMisc('$90.00', '$10.00', '$86.50');
+    assertSelectedPaymentMethod('checkmo', 0);
+    cy.wait(5000);
+    placeOrder();
+    assertOrderConfirmationCommonDetails(customerBillingAddress);
+    assertOrderConfirmationShippingDetails(customerShippingAddress);
+    assertOrderConfirmationBillingDetails(customerBillingAddress);
+    assertOrderConfirmationShippingMethod(customerShippingAddress);
+    /**
+     * TODO - when /customer/order-details page will be ready
+     * Redirect to /order-details?orderRef={ORDER_TOKEN}
+     * Confirm that elements similar to orderConfirmation page present (not exactly the same, separate assert needed)
+     */
+    /**
+     * TODO - when /customer/account page will be ready
+     * Redirect to /customer/account
+     * Confirm that new order is visible in Recent Orders section of account dashboard
+     */
+    /**
+     * TODO - when /customer/orders page will be ready
+     * Redirect to /customer/orders
+     * Confirm that new order is visible on Orders page
+     */
+
+    // Obtain order reference from URL and visit order details page
+
+    cy.url().then((url) => {
+      const orderRef = url.split('?')[1];
+      cy.visit('/order-details?' + orderRef)
+    })
+
+    // CANCEL ORDER
+    cy.get(fields.cancelButton).should('exist');
+    cy.get(fields.cancelButton).click();
+
+    cy.get(fields.cancellationReasonsSelector).select('1');
+    cy.get(fields.cancellationReasonsSelector).should('have.value', '1');
+
+    cy.get(fields.submitCancelOrderButton).click();
+
+    cy.get('.dropin-header-container__title', {timeout: 3000})
```

Comment:
In general, we should avoid adding timeouts to specific commands unless the default is less than this and would cause a failure.

https://docs.cypress.io/api/commands/get has defaultCommandTimeout as default.